lib/scripts/isWithinViewport.js

```diff
@@ -26,14 +26,23 @@ let isWithinViewport = function (elements) {
                 height: dde.clientHeight
             }
 
-            isWithinViewport = isWithinViewport &&
+            // if static, we can ignore its position, height and width attribute
```

Comment:
I guess we need to determine the position in the document then and check if it is within it, right?